bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -898,6 +898,71 @@ impl dvm_rpc_runtime_api::ConvertTransaction<OpaqueExtrinsic> for TransactionCon
 	}
 }
 
+const PARCEL: &[u8] = &[
```

Comment:
Put this inside test function if this is only used for test?

Reply:
Not only for the test but also [here](https://github.com/darwinia-network/darwinia-common/pull/708/files/a87307ede483ea3ac52a2f5d6545265cc5e92791#diff-d76e6add241f9f0be48db383dc3c10d6cd079d48703a6972e3ef9f3e64e29d80R984)

Reply:
How about put the  `PARCEL` into `darwinia_ethereum_relay::migration()`? It looks a bit wired to see this test here.

```rs
darwinia_ethereum_relay::migration::migrate() {
        let PARCEL = ...
}
```